qanary_commons/src/main/java/eu/wdaqua/qanary/communications/RestTemplateWithProcessId.java

```diff
@@ -16,10 +15,11 @@
  * call to the API-Request header.
  */
 @Service
-@ConditionalOnExpression("'${rest.template.setting}' == 'A' or '${rest.template.setting}' == 'C'") // A = Both, C = ProcessIdRestTemplate
+@ConditionalOnExpression("'${rest.template.setting}' == 'A' or '${rest.template.setting}' == 'C'")
```

Comment:
@dschiese Why use "A" and "C" here when you could also use the actual names "Both" and "ProcessIdRestTemplate"?